receiver/tlscheckreceiver/documentation.md

```diff
@@ -0,0 +1,59 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# tlscheckreceiver
+
+## Default Metrics
+
+The following metrics are emitted by default. Each of them can be disabled by applying the following configuration:
+
+```yaml
+metrics:
+  <metric_name>:
+    enabled: false
+```
+
+### tlscheck.days_left
+
+Days until X.509 certificate expiration.
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| s | Gauge | Int |
```

Comment:
see my other comment on units - days vs. seconds. It's not part of UCUM, but seems like `d` might work for days.